Challenge01-AzureAPIforFHIR/ReadMe.md

```diff
@@ -236,6 +245,10 @@ Below are some common setup issues you might run into with possible resolution.
 * **Windows Terminal running PowerShell**: This is a known issue.
 Connect-AzureAD -TenantID <tenantid> does not open the Auth Login popup. This happens in both admin mode and non-admin mode.
 
+* **Unable to find type [System.Web.Security.Membership]**: This can occurr when running `Create-FhirServerSamplesEnvironment.ps1` because the `System.Web.Security.Membership` module is not part of PS Core on `*nix` platforms. You can overcome this by providing a password using this argument:
+  ```powershell
+  -adminPassword $(ConvertTo-SecureString -AsPlainText -Force '<Some Password Here>')
+  ```
 ***
 
```

Comment:
There is a scenario I think we should call out when a user on a *nix machine installs the `AzureAD` Powershell module and then sees the instructions for the `AzureAD.Standard.Preview` module and proceeds to install that. If the user follows that path, they will get:

`Assembly with same name is already loaded`

The solution to this is to uninstall the `AzureAd` module and restart Powershell:

`Uninstall-Module AzureAD`

Reply:
I did run into this too, so maybe its not as uncommon as I expected

Reply:
Interesting, I did not but it does follow that you'd need to run these. Let me go ahead and add this to the document. Thank you for pointing it out!

FYI, I am on Day 2 of powershell so I'm learning as I go :D

Reply:
Done!